build.cake

```diff
@@ -553,14 +553,37 @@ Task("_cg-uwp-run-tests")
 
             NUnit3(new [] { UWP_TEST_LIBRARY }, settings);
         }
-        finally
+        catch
         {
+            SetEnvironmentVariables();
+            PrintEnvironmentVariables();
+            throw;
+        }
+        finally
+        { 
             try
             {
                 process?.Kill();
             }
             catch{}
         }
+
+        SetEnvironmentVariables();
+
+        void SetEnvironmentVariables()
+        {
+            var doc = new System.Xml.XmlDocument();
+            doc.Load("TestResult.xml");
+            var root = doc.DocumentElement;
+            String failedTests = root.GetAttribute("failed");
+            String total = root.GetAttribute("total");
+            
+            Information("failedTests: {0}", failedTests);
+            Information("totalTests: {0}", total);
+
+            System.Environment.SetEnvironmentVariable("failedTests", failedTests);
```

Comment:
@rmarinho does this work?

Reply:
Seems not is setting empty valus not sure why